src/translationmanager.js

```diff
@@ -0,0 +1,31 @@
+import { format } from './utils';
+import getTranslationsJSON from './translations';
+
+export default class TranslationManager {
+    constructor(instance) {
+        this.instance = instance;
+        this.language = this.instance.language;
```

Comment:
```suggestion
    constructor(language) {
        this.language = language;
```

`instance` is not used in this class.